transport.go

```diff
@@ -130,6 +134,18 @@ func NewPCTransport(params PCTransportParams) (*PCTransport, error) {
 		i.Add(sdkinterceptor.NewRTTInterceptorFactory(t.handleRTTUpdate))
 	}
 
+	var onXRRtt func(rtt uint32)
+	if params.IsSender {
+		// publisher only responds to XR request for sfu to measure RTT
+		onXRRtt = func(rtt uint32) {}
+	} else {
+		onXRRtt = func(rtt uint32) {
+			t.rttFromXR.Store(true)
+			t.SetRTT(rtt)
```

Comment:
Should be calling helper method instead.

```suggestion
			t.setRTT(rtt)
```